contrib-docs/community-guidelines/index.md

```diff
@@ -1,30 +1,30 @@
+👋 Welcome to MDN Web Docs on GitHub! MDN Web Docs is an open-source, collaborative project that documents web platform technologies, including [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML), [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS), [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), and [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API). We also provide extensive [learning resources](https://developer.mozilla.org/en-US/docs/Learn) for early-stage developers and students.
 
 ## Ways to contribute
 
-- (Learn Forum)[linktolearnforum] - help users in our learn forum, see the guidelines on how to here
-- (Translated Content)[linktotranslatedcontent] - help translate MDN Web Docs into one of our active locales (list locales)
-- (Help with updating content or platform improvements by contributing to our repositories)[listofrepositories]
-	- issues
-	- pr reviews
+- [Fixing MDN Web Docs content bugs](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Fixing_MDN_content_bugs)
+- [Reviewing MDN Web Docs edits](https://github.com/mdn/content/blob/main/REVIEWING.md)
+- [Help beginners to learn on MDN Web Docs](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Help_beginners)
+- [Contribute to MDN Web Docs interactive examples](https://github.com/mdn/interactive-examples/blob/main/CONTRIBUTING.md)
+- [Help translate MDN Web Docs](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Localize)
+- [Help fix known platform issues](https://github.com/mdn/yari/issues)
+- [Help us keep browser compatibility data up to date.](https://github.com/mdn/browser-compat-data)
 
 ## Quick Links
 
-- Quick Start Guide. If you are familiar with git and GitHub _(how to run repos should be within each repos readme)_
-- Get Started on GitHub. For those who want to contribute to content or the platform and are less familiar with git or GitHub, start here.
+- [If you are familiar with Git and GitHub](./contributing/getting-started/general/index.md)
+- [Get Started with Git and GitHub](./contributing/getting-started/beginners/index.md)
+- [Overview of our other repositories](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Where_is_everything)
 
 ## Code of conduct
 
-_()_
+By participating in and contributing to our projects and discussions, you acknowledge that you have read and agree to the [Mozilla community participation guidelines](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md).
+
+## Get in touch
+
+You can communicate with the MDN Web Docs team and community through our [Matrix channel](https://chat.mozilla.org/#/room/#mdn:mozilla.org), forums on [Discourse](https://discourse.mozilla.org/c/mdn/236), and discussions on [GitHub](https://github.com/mdn/mdn-community/discussions).
 
-## Get in touch
\ No newline at end of file
+- You can also drop us an email at <mdn-admins@mozilla.org>.
```

Comment:
I was not sure if we still wanted to include this. Does anyone manage this email address?

Reply:
No we should just remove this line - I wonder how many people have emailed 😳